doc/cli/vast-import-zeek.md

```diff
@@ -1,5 +1,7 @@
 The `import zeek` command consumes [Zeek](https://zeek.org) logs in
-tab-separated value (TSV) style.
+tab-separated value (TSV) style, and the `import zeek-json` command consumes
+Zeek logs as [line-delimited
+JSON](https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON) objects.
```

Comment:
I think we should mention that there 3 types of Zeek logs:

1. TSV
2. Stock NDJSON
3. Streaming NDJSON